urlColorsServiceWorker.js

```diff
@@ -0,0 +1,117 @@
+let snoozeTimeout;
+
+const updateValue = function(property, value) {
+  chrome.storage.local.set({[property]: value}, function() {
+    console.log(`Updated ${property} to: `, value);
+  });
+};
+
+const handleSnooze = (snoozeTime) => {
+  clearTimeout(snoozeTimeout);
+  const diffInTime = snoozeTime - Date.now();
+  if (diffInTime > 0) {
+    snoozeTimeout = setTimeout(function() {
```

Comment:
Sometimes you are using `function()` and sometimes you are using `() =>`

Not critical but you could be more consistent.